addons/interaction/initSettings.sqf

```diff
@@ -0,0 +1,32 @@
+[
+    QGVAR(enableTeamManagement), "CHECKBOX",
+    [LSTRING(EnableTeamManagement_DisplayName), LSTRING(EnableTeamManagement_Description)],
+    format ["ACE %1", LLSTRING(DisplayName)],
+    true,
+    true
+] call CBA_fnc_addSetting;
+
+[
+    QGVAR(enableMagazinePassing), "CHECKBOX",
+    LSTRING(PassMagazineSetting),
+    format ["ACE %1", LLSTRING(DisplayName)],
+    true
+] call CBA_fnc_addSetting;
+
+[
+    QGVAR(disableNegativeRating), "CHECKBOX",
+    [LSTRING(DisableNegativeRating_DisplayName), LSTRING(DisableNegativeRating_Description)],
+    format ["ACE %1", LLSTRING(DisplayName)],
+    false,
+    true,
+    {[QGVAR(disableNegativeRating), _this] call EFUNC(common,cbaSettings_settingChanged)},
+    true
+] call CBA_fnc_addSetting;
+
+[
+    QGVAR(weaponAttachments), "CHECKBOX",
```

Comment:
Maybe `enableAttachmentActions` or `weaponAttachmentActions`.

Reply:
How about `weaponAttachmentsEnabled`? `Actions` is like to be meant here, only actions can be in `interaction` component.

Reply:
I think `enableWeaponAttachments` to match the naming of the team management and magazine passing settings.

Reply:
imho `enable...` mostly suits for function name and `...Enabled` for bool variables or event name. There are many ACE settings with both formats. Which is preferred?

Reply:
I'd say `enable...` for consistency within the `interaction` component.